values/prod/spacerace/api-read.yaml

```diff
@@ -1,3 +1,11 @@
+cache:
+  enabled: true
+  proxyurl: http://servicename:1234/rpc/v0
+  jwtbase64: x
```

Comment:
Can we refer to Secret entities instead of using full configmaps somehow? I really discourage the idea of setting secret vars in CM